docs/multicluster/quick-start.md

```diff
@@ -0,0 +1,174 @@
+# Antrea Multi-cluster Quick Start
+
+In this quick start guide, we will set up an Antrea Multi-cluster ClusterSet
+with two clusters. One cluster will serve as the leader of the ClusterSet, and
+meanwhile also join as a member cluster; another cluster will be a member only.
+
+The diagram below shows the two clusters and the ClusterSet to be created (for
+simplicity, the diagram just shows two Nodes for each cluster).
+
+<img src="assets/sample-clusterset.svg" width="1000" alt="Antrea Multi-cluster Example ClusterSet">
+
+## Preparation
+
+We assume an Antrea version >= `v1.7.0` is used in this guide, and the Antrea
+version is set to an environment variable `TAG`. For example, the following
+command sets the Antrea version to `v1.7.0`.
+
+```bash
+export TAG=v1.7.0
+```
+
+To use the latest version of Antrea Multi-cluster from the Antrea main branch,
+you can change the YAML manifest path to: `https://github.com/antrea-io/antrea/tree/main/multicluster/build/yamls/`
+when applying or downloading an Antrea YAML manifest.
+
+Antrea must be deployed in both cluster A and cluster B. To configure Antrea
+Multi-cluster Gateways, the `Multicluster` feature of `antrea-agent` must be
+feature enabled in both cluster A and B. Multi-cluster Gateways are required for
+routing multi-cluster Service traffic across the member clusters. Set the
+following configuration parameters in `antrea-agent.conf` of the Antrea
+deployment manifest to enable the `Multicluster` feature:
+
+```yaml
+antrea-agent.conf: |
+...
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enable: true
+    namespace: ""
+```
+
+## Set up Leader and Member in Cluster A
+
+### Step 1 - deploy Antrea Multi-cluster Controllers for leader and member
+
+Run the following commands to deploy Multi-cluster Controller for the leader
+into Namespace `antrea-multicluster` (Namespace `antrea-multicluster` will be
+created by the commands), and  Multi-cluster Controller for the member into
+Namepsace `kube-system`.
+
+```bash
+$kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-global.yml
+$kubectl create ns antrea-multicluster
+$curl -L https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-namespaced.yml > antrea-multicluster-leader-namespaced.yml
+$sed 's/changeme/antrea-multicluster/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
+$kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-member.yml
+```
+
```

Comment:
It might be good to add a command to check the deployment results that can show both leader & member mc-controller Deployments and Pods.

Does something like work: `kc get all -A -l="app=xyz"`

What should be the "xyz".

Reply:
sure, I will add it, it can be `kubectl get all -A -l="component=antrea-mc-controller"`, should I add a sample output?

Reply:
Done